### ISSUE TITLE
[NBug] InvalidArgument=Value of '0' is not valid for 'index'.Par...

### ISSUE THREAD

**Issue by talalmog8** [gitextensions/gitextensions#6964]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.ArgumentOutOfRangeException: InvalidArgument=Value of '0' is not valid for 'index'.
Parameter name: index
   at System.Windows.Forms.ListView.SelectedListViewItemCollection.get_Item(Int32 index)
   at GitUI.CommandsDialogs.SettingsDialog.Pages.FormChooseTranslation.lvTranslations_ItemActivate(Object sender, EventArgs e)
   at System.Windows.Forms.ListView.OnItemActivate(EventArgs e)
   at System.Windows.Forms.ListView.WmReflectNotify(Message& m)
   at System.Windows.Forms.ListView.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Environment
- Git Extensions 3.1.1.6049
- Build 2f872105b4eb42421d9fdafd68e90a9e27d20d96
- Git 
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.8.3801.0
- DPI 96dpi (no scaling)

**Comment by gerhardol** [comment]
Duplicate of #6680

**Comment by RussKie** [comment]
@talalmog8 how did this happen?

@gerhardol it looks similar, but the the callstack is slightly different

**Comment by gerhardol** [comment]
Duplicate of #6726 
Corrected in 3.2

**Comment by RussKie** [comment]
Thanks Gerhard 👍 
I was looking at the latest code and couldn't see how it could fail.
